src/styles/blocks/catalog.scss

```diff
@@ -2,24 +2,32 @@
   display: flex;
   flex-wrap: wrap;
 
-  max-width: 944px;
+  max-width: $catalog-width;
   margin: 0 auto;
-  padding: 50px 0;
+  padding: $catalog-padding-y 0;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
 
-    width: 200px;
+    width: $card-width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition-duration: $trans-dur;
+    transition-timing-function: $smooth;
+
     &:not(:nth-child(4n)) {
-      margin-right: 48px;
+      margin-right: $card-space;
```

Comment:
what's card-space? not clear to me what does this variable mean - try sth more descriptive

Reply:
fixed